README.md

```diff
@@ -23,6 +23,9 @@ For local builds, run `make` which will build the binary, run unit tests and lin
 
 ## Releasing
 
+> Go 1.16 or later is required to build for newer architectures, such as `darwin-arm64`.
+> Go 1.17 is recommended.
+
```

Comment:
This is enforced by `go.mod` file change.

Reply:
I thought we needed some additional clarity about that, but will remove.